syntax = "proto3";

package service;

option java_package = "service.proto"; 
option java_multiple_files = true; 

service CountLettersService {
    rpc CountLetters (InputString) returns (LetterNumber);
}

message InputString {
  string input_string = 1;
}

message LetterNumber {
  int32 letter_number = 1;
}